src/cli/main.py

```
@@ -35,16 +35,17 @@
     "NL キャッシュは使用・作成しません。"
 )
 FETCH_NOTE_TO_CLIENT_FILTER = (
-    "--to は取得後のクライアント側日付フィルタであり、JVOpen の終端ではありません。"
+    "--to は取得後のクライアント側日付フィルタです。"
+    "範囲形式を使える dataspec では JVOpen の終端にもなります。"
 )
 FETCH_NOTE_TO_SINGLE_OPEN = (
-    "option=1/2 では --to は JVOpen の終端にならず、"
-    "狭めてもサーバからのダウンロード量は減りません。"
+    "この dataspec は範囲形式 fromtime を使えないため、--to を狭めても"
+    "サーバからのダウンロード量は減りません（終了時刻を指定できない種別です）。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)